src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -95,7 +95,7 @@ export default (opts) => {
     ),
     incrementsLoader: gravityLoader("increments"),
     inquiryRequestQuestionsLoader: gravityLoader(`inquiry_request_questions`),
-    matchArtistsLoader: gravityLoader("match/artists"),
+    matchArtistsLoader: gravityLoader("match/artists", {}, { headers: true }),
```

Comment:
Any existing invocations of `matchArtistsLoader` would need to be updated to work with the new signature (returning `body` and `headers` instead of the data directly).

Reply:
Ooo good call. Don't think I had fully registered exactly what [was happening](https://github.com/artsy/metaphysics/blob/3e9e871e84e18ee178d86c2fc522fe0c4eb2953d/src/lib/loaders/api/loader_without_authentication_factory.ts#L101-L102) when passing in the `headers: true` option. 

Fixed in 38d9e46

Reply:
(and the test mock!)